smart-contracts/test/Lock/timeMachine.js

```diff
@@ -86,4 +87,13 @@ contract('Lock / timeMachine', accounts => {
       assert.equal(tx.logs[0].event, 'ExpirationChanged')
     })
   })
+
+  describe('failures', async () => {
+    it('should not work for a non-existant key', async () => {
+      await shouldFail(
```

Comment:
Looks good - but I'm hoping to deprecate our shouldFail helper.  Let's start migrating to truffle-assertions:

```
const { reverts } = require('truffle-assertions')
...
await reverts(lock.timeMachine(...), 'NON_EXISTENT_KEY')
```
```

Reply:
sure, I'll migrate this one now and repush. If it passes I'll merge.

Reply:
Sounds good. btw it's a helpful library, they have a few other calls like eventEmitted that could be useful for us https://github.com/rkalis/truffle-assertions